src/util/promiseSequence.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Provides a mechanism to create a list of promises one at a time.
```

Comment:
```suggestion
 * Provides a mechanism to invoke a list of promises one at a time.
 * This can used by a {@link PromiseSequenceFunc} instead of Promise.all
```